tests/AiDotNet.Tensors.Benchmarks/BaselineRunners/py/run_direct_ptx_rglru_competitors.py

```
@@ -0,0 +1,212 @@
+def percentile(values, q):
+    ordered = sorted(values)
+    position = (len(ordered) - 1) * q
+    lower = int(position)
+    upper = min(lower + 1, len(ordered) - 1)
+    return ordered[lower] + (ordered[upper] - ordered[lower]) * (position - lower)
+
+
+def query_gpu_status():
+    result = subprocess.run(
+        ["nvidia-smi", "--query-gpu=utilization.gpu,memory.used,temperature.gpu",
+         "--format=csv,noheader,nounits"],
```

Ruff (0.16.0)

[error] Starting a process with a partial executable path

(S607)